CMakeLists.txt

```diff
@@ -38,17 +40,50 @@ if(ARGH_MASTER_PROJECT)
 	install(TARGETS argh EXPORT arghTargets)
 
 	include(GNUInstallDirs)
+	string(REPLACE "/${CMAKE_LIBRARY_ARCHITECTURE}" "" CMAKE_INSTALL_LIBDIR_ARCHIND "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
I think in general it is better to not start user variables with ``CMAKE_``, as they look like CMake built-in ones, but they are not.

Reply:
Good point. And in fact it'd be nice to have a builtin CMake one doing the same. The change into GNUInstallDirs.cmake should be trivial, do we have any volunteers who gonna implement it and send a PR there?